src/modules/organizations/components/organization-layout.jsx

```diff
@@ -1,30 +1,43 @@
 import React from 'react';
-import { Link } from 'react-router';
+import { IndexLink, Link } from 'react-router';
+import { config } from '../../../constants/config';
 
-const OrganizationLayout = ({ children, navItems, organizationId }) => (
+const OrganizationLayout = ({ children, navItems, organizationId, organizationSlug }) => (
   <div className="organization-layout">
     <nav className="organization-layout__nav">
       <ul className="nav-list">
         <li><div className="nav-list-header">Organization #{organizationId}</div></li>
         <li>
-          <Link
-            to={`/organizations/${organizationId}`}
+          <a
             className="view-org-button"
+            href={`${config.zooniverseURL}organizations/${organizationSlug}`}
```

Comment:
I'm getting a 404 response for the URL generated here, is that the desired result at this point?

![image](https://user-images.githubusercontent.com/12473119/29280958-c393ed82-80e2-11e7-89f1-420412c6a1f1.png)